pulse_actions/run_time_config.json

```diff
@@ -1,4 +1,4 @@
 {
-    "exchange": "exchange/treeherder/v1/job-actions",
-    "topic": "buildbot.ash.#"
+    "exchange": "exchange/build/normalized",
+    "topic": "unittest.mozilla-inbound.#"
```

Comment:
You might want to ask in the dev.tools mailing list for suggestions.
I believe we can spawn a process per consumer if that is the right of doing it.
However, I'm not experienced with parallelization.